src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+
+import './GenreSelect.css';
+
+type GenreSelectType = {
```

Comment:
I guess it is better to use `interface` instead. type and interface can be used to define props in a React component, but they have some differences in functionality. interface is useful for defining simple props that can be extended or reused, while type is useful for defining more complex types in your props